app/serializers/api/v1/task_serializer.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class TaskSerializer < ActiveModel::Serializer
+      attributes :id, :name, :description, :status
+      attribute :deadline_at do
+        object.deadline_at.strftime("%d-%m-%Y %H:%M")
```

Comment:
Is it return in format year-month-day?

Reply:
it is returned in format year-month-day and hours-minutes
like "26-11-2023 15:00"

Reply:
Please return only date